### ISSUE TITLE
Error with dependency when running `npm run start`

### ISSUE THREAD

**Issue by LucasDachman** [collective-soundworks/soundworks-template#4]
```
$ npm run start

> soundworks-template@2.2.0 start /Users/lucas/summer-2019/soundworks-template-1
> node bin/runner --start


/Users/lucas/summer-2019/soundworks-template-1/node_modules/soundworks/node_modules/uws/uws.js:44
})();
  ^
Error: Compilation of µWebSockets has failed and there is no pre-compiled binary available for your system. Please install a supported C++11 compiler and reinstall the module 'uws'.
    at /Users/lucas/summer-2019/soundworks-template-1/node_modules/soundworks/node_modules/uws/uws.js:40:19
    at Object.<anonymous> (/Users/lucas/summer-2019/soundworks-template-1/node_modules/soundworks/node_modules/uws/uws.js:44:3)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
    at Module.load (internal/modules/cjs/loader.js:685:32)
    at Function.Module._load (internal/modules/cjs/loader.js:620:12)
    at Module.require (internal/modules/cjs/loader.js:723:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> (/Users/lucas/summer-2019/soundworks-template-1/node_modules/soundworks/server/services/RawSocket.js:4:1)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
```

I've tried removing `node_modules` and reinstalling but that didn't help

**Comment by b-ma** [comment]
Hi,

I just released a new version (v2.2.1) that removes the dependency to `uws`, can you retry and tell me if it works for you?

Thanks,

**Comment by LucasDachman** [comment]
Works now thanks!
